### PR TITLE
Implemet conversion of XML namespace imports

### PR DESCRIPTION
**Pull request by Noah1989**
[Link to issue(s) this covers](https://github.com/icsharpcode/CodeConverter/issues/836)

This PR implements the approach suggested by me in that issue. I decided to keep it at one helper class per file (if needed) to keep the scope where imports apply the same as in VB.

### Problem
[XML import statements](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/statements/imports-statement-xml-namespace#example-1) are not converted.

### Solution
* This solution tries to mimic the behavior of VB XML namespace imports.
* If any XML namespace imports are present, a helper class is generated per file, which contains the namespace definitions. This is in line with how they apply in VB. This also keeps them at the top of the file and allows for changing them easily after conversion.
* The document name is included in the generated helper class name in order to avoid duplicate names while staying deterministic. A using directive is generated to provide a shorter (common) alias.
* When converting XML names with prefixes, the namespace definitions from the helper class are used.
* The helper class is also used to manage xmlns attributes. Whenever a XContainer is created, they are added, and any dupicate attributes from the child elements are removed. This is in line with what the helper code generated by VB does.
* [x] At least one test covering the code changed

### CODE REVIEW

**Comment by GrahamTheCoder** [comment]
I'm recovering from quite a bad bout of covid at the moment so don't have the energy to fully understand and give this the in depth review I was hoping to. But from just glancing over it, it's very clearly a big improvement, and looks like good quality code. Thanks a lot for your efforts!

File: CodeConverter/CSharp/DeclarationNodeVisitor.cs [hunk 1]
```diff
@@ -1607,5 +1612,34 @@ public override async Task<CSharpSyntaxNode> VisitTypeConstraint(VBSyntax.TypeCo
...
+                // namespace alias
+                return SyntaxFactory.IdentifierName(node.LocalName.ValueText);
+            } else {
+                // Having this case in VB would cause error BC31187: Namespace declaration must start with 'xmlns'
+                throw new NotImplementedException($"Cannot convert non-xmlns attribute in XML namespace import").WithNodeInformation(node);
+            }
+        }
+
+        public override async Task<CSharpSyntaxNode> VisitXmlString(VBSyntax.XmlStringSyntax node) =>
+            CommonConversions.Literal(node.TextTokens.Aggregate("", (a, b) => a + LiteralConversions.EscapeVerbatimQuotes(b.Text)));
```
Comment (GrahamTheCoder) [inline CodeConverter/CSharp/DeclarationNodeVisitor.cs:1643]: Not sure of the likely inputs here myself, but if there could be quite a few strings and especially if they could be big, consider StringBuilder

Comment (Noah1989) [reply CodeConverter/CSharp/DeclarationNodeVisitor.cs:1643]: Agreed. I guess String.Join could also work, right?

Comment (GrahamTheCoder) [reply CodeConverter/CSharp/DeclarationNodeVisitor.cs:1643]: Yep, that'd probably be slightly more readable too actually 👍

**Pull request merged**